ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_dialog/.content.xml

```diff
@@ -128,6 +128,20 @@
                                                     fieldDescription="You can specify a Document of Record Bind Reference to link an adaptive form field with a document of record field. When you fill any value in the linked field of an adaptive form, the value is also displayed in the linked field of the corresponding document of a record."
                                                     fieldLabel="Document of Record bind reference"
                                                     name="./dorBindRef"/>
+                                            <unboundFormElement
+                                              jcr:primaryType="nt:unstructured"
+                                              sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                              granite:class="cmp-adaptiveform-base__unboundformelement"
+                                              name="./unboundFormElement"
+                                              text="Mark as Unbound Form Element"
```

Comment:
Can you add some description or help text to this ? Please sync with khushwant on this

Reply:
Will raise a separate ticket and discuss with khushwant